index.html

```diff
@@ -10,7 +10,25 @@
 </head>
 
 <body>
-    <h1>Hello</h1> 
+    <main id="main">
+        <div>
+        <h1 id="title">Dr. Norman Borlaug</h1>
+        <div id="img-div">
+            <img src="" alt="" id="image">
+            <div id="img-caption">
+                <h1>
+                <p>Image the tribute</p>
+                </h1>
+            </div>
+        </div>
+        <div id="tribute-info">
+            <h2>Information the tribute</h2>
+        </div>
+        <a href="" id="tribute-link" target="_blank"></a>
+        </div>
+
+    </main>
+    <h1>Microverse trials</h1>
```

Comment:
Hi, @pradmon!! 👋
- Here you have made a good suggestion politely using reasoning. 👍 

- [ ] Kindly use emoticon (😄) so that your review looks human-friendly.

- [ ] Kindly use your reasoning `why` in a precise way so that students can easily understand.
  - Here you have claimed it is `a good programming practice`. But why? I would suggest you can add `this change would be highly regarded as good programming practice cause this increases the accessibility of a website.`